apps/api/src/helpers/csv.ts

```diff
@@ -0,0 +1,22 @@
+import { stringify } from 'csv-stringify';
```

Comment:
Nest is modular, so it might be a good idea to put it under `src/common`. Also, `helpers` could be singular